src/js/react/common/Card.tsx

```diff
@@ -78,7 +80,7 @@ function CardItem({
       }
 
       <div className="card__text">
-        <HeadingTag className="card__title">
+        <HeadingTag className="card__title" {...langAttribute}>
           <a href={cardUrl} className="card__link" {...cardUrlExternal && { 'data-is-external': 'true' }} rel="bookmark">
             <span>{ cardTitle }</span>
```

Comment:
The titles can be in this format where the:
```
<h3>
  <a>
    <span>
      <span>Ilmoituksen otsikko</span>
      <span>(2 tehtävää)</span>
    </span>
  </a>
</h3>
```

The "(2 tehtävää)" span is translated to the UI language, so we should wrap the first span with the langAttribute instead of the whole heading-element.

Reply:
Fixed.